src/Dfc.CourseDirectory.Web/Controllers/UnregulatedCoursesController.cs

```diff
@@ -268,7 +268,7 @@ public async Task<IActionResult> ZCodeNotKnown([FromQuery] ZCodeNotKnownRequestM
                 model.Url = Request.GetDisplayUrl();
                 model.PageParamName = _larsSearchSettings.PageParamName;
                 model.ItemsPerPage = _larsSearchSettings.ItemsPerPage;
-                model.TotalCount = result.Value.ODataCount ?? 0;
+                model.TotalCount = result.Value.ODataCount / model.ItemsPerPage ?? 0;
```

Comment:
This isn't correct - `TotalCount` should be total number of items not total number of pages (calculation for that is incorrect at any rate)

Reply:
now calculating before invoking VC